opensrp-app/src/main/java/org/smartregister/util/FormUtils.java

```diff
@@ -1077,8 +1078,18 @@ private String readFileFromAssetsFolder(String fileName) {
     public JSONObject getFormJson(String formIdentity) {
         if (mContext != null) {
             try {
-                InputStream inputStream = mContext.getApplicationContext().getAssets()
-                        .open("json" + ".form/" + formIdentity + ".json");
+                String locale = mContext.getResources().getConfiguration().locale.getLanguage();
+                locale = locale.equalsIgnoreCase("en") ? "" : "_"+locale;
```

Comment:
We could add a check here for the flow when the default of the device is another language but the form that is local specific for that doesn't exist